toolbox

```diff
@@ -1473,6 +1478,9 @@ usage()
     echo "   or: toolbox [-y | --assumeyes]"
     echo "               rmi [-a | --all]"
     echo "                   [-f | --force] [<image> ...]"
+    echo "   or: toolbox [-v | --verbose]"
+    echo "               run [-c | --container <name>]"
+    echo "                   [-r | --release <release>]"
```

Comment:
We should at least update `completion/bash/toolbox` with the command and its options. Would be even nicer to have completion for the command.